firmware/application/apps/ui_about_simple.cpp

```diff
@@ -33,6 +33,7 @@ void AboutView::update() {
                 console.writeln("heurist1,intoxsick,ckuethe");
                 console.writeln("notpike,jLynx,zigad");
                 console.writeln("MichalLeonBorsuk,jimilinuxguy");
+                console.writeln("kallanreed");

```

Comment:
Man, I'm astonished we didn't add you. What's @eried doing ? ;-p
Can you please also add @bernd-herzog ?

Reply:
Added